src/main/xerus/monstercat/Settings.kt

```diff
@@ -23,6 +24,8 @@ object Settings : SettingsNode("xerus/monsterutilities") {
 	val PLAYERSCROLLSENSITIVITY = create("playerSeekbarScrollSensitivity", 6.0)
 	val PLAYERSEEKBARHEIGHT = create("playerSeekbarHeight", 8.0)
 	val ENABLEEQUALIZER = create("equalizerEnabled", false)
+	val EXPORTCURRENTTITLE = create("exportPlayerTitle", false)
```

Comment:
No need to have two options. I'd simply set the other option to be empty by default, and only when it is non-empty it will export the value.

Reply:
Good idea

Reply:
a899097197c5150bd197d6f508cd6460878d6a05